Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -59,7 +75,8 @@ public struct Configuration: Hashable {
                  ruleList: RuleList = masterRuleList,
                  configuredRules: [Rule]? = nil,
                  swiftlintVersion: String? = nil,
-                 cachePath: String? = nil) {
+                 cachePath: String? = nil,
+                 indentationMode: IndentationMode = .spaces(count: 4)) {
```

Comment:
I'm not sure if this is a good place to set the default value - the default value is duplicated in a few places like the the Configuration parser, and AutoCorrectCommand, which isn't ideal.

Reply:
Why not have a `static var default = .spaces(count: 4)`?